web/application/controllers/Apis.php

```diff
@@ -1035,52 +1036,124 @@ public function airqoPlacesJobUpdate()
+//                 //     CURLOPT_FOLLOWLOCATION => true,
+//                 //     CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+//                 //     CURLOPT_CUSTOMREQUEST => "POST",
+//                 //     CURLOPT_POSTFIELDS => $data,
+//                 //     CURLOPT_HTTPHEADER => array(
+//                 //         "Content-Type: application/json"
+//                 //     ),
+//                 // ));
+
+//                 // $result = curl_exec($curl);
+
+//                 // curl_close($curl);
+//                 // $response = json_decode($result, true);
+                
+//                 $state      = $this->ApisModel->stateOk();
+//                 $state_name = "success";
+//                 $state_code = 100;
+//                 $message    = "Sucessful";
+//                 $debug      = "API Config OK";
+//                 echo  $this->ApisModel->api_response($response, $state, $state_name, $state_code, $message, $debug);
+//             } else {
+//                 echo $validate;
+//             }
+//         }
+//     }
+
+    public function placeForecast()
     {   
         $response = array();
-        $this->ApisModel->init();
-        $this->form_validation->set_rules('lat', 'Place latitude', 'trim|required');
-        $this->form_validation->set_rules('lng', 'Place longitude', 'trim|required');
+        $this->ApisModel->init(); 
+        $this->form_validation->set_rules('channel', 'Channel ID', 'trim|required');
         $this->form_validation->set_rules('api', 'API', 'trim|required');
         if ($this->form_validation->run() == false) {
             echo $this->ApisModel->api_error();
         } else {
-            $lat = $this->ApisModel->escape($this->input->post("lat"));
-            $lng = $this->ApisModel->escape($this->input->post("lng"));
+            $channel = $this->ApisModel->escape($this->input->post("channel"));
             $api = $this->ApisModel->escape($this->input->post("api"));
             $validate = $this->ApisModel->validateAPI($api);
             if ($validate == $this->ApisModel->stateOk()) {
 
-                $url = "https://ml-service-dot-airqo-250220.appspot.com/api/v1/predict/";
-                $ch = curl_init($url);
-
-                $lat = $this->ApisModel->escape($this->input->post("lat"));
-                $lng = $this->ApisModel->escape($this->input->post("lng"));
-
                 date_default_timezone_set('Africa/Kampala');
                 $now  =  date("Y-m-d H:i:s");
                 $time = strtotime($now);
-                $time = $time - (60*60);
-                $selected_datetime = date("Y-m-d H:i", $time);
+                // $time = $time - (60*60);
+                // $selected_datetime = date("Y-m-d H:i", $time);
+                $selected_time = time();
+
+                $ch = curl_init();
+
+                curl_setopt($ch, CURLOPT_URL, "http://34.78.78.202:31009/api/v2/predict/".$channel."/" . $selected_time);
```

Comment:
- could we use the urls instead of these ones which expose our ports? That is: staging-platform.airqo.net or platform.airqo.net?
- 
- How do we separate our development environment here?